<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="tableView"/>
      <widget class="QTableView" name="tableView_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionData"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSettings"/>
   <addaction name="actionNormalize"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/ui/res/folder.ico</normaloff>:/res/ui/res/folder.ico</iconset>
   </property>
   <property name="text">
    <string>Load data file</string>
   </property>
  </action>
  <action name="actionData">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/ui/res/settings.png</normaloff>:/res/ui/res/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Normalization Settings</string>
   </property>
  </action>
  <action name="actionNormalize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/ui/res/norm.png</normaloff>:/res/ui/res/norm.png</iconset>
   </property>
   <property name="text">
    <string>Normalize</string>
   </property>
   <property name="toolTip">
    <string>Normalize dataset</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
